clarification on connected apps on business groups (#128)
mulesoft/docs-access-management: modules/ROOT/pages/connected-apps-org-admin.adoc

--
Applications created by users in the same organization are marked as *internal apps* and do not need to be whitelisted manually.
--

> == Master organization and business groups

Reviewer: ```suggestion
== Master Organization and Business Groups
```

> Connected Apps are at the Master Organization level, not at the Business Group(Sub Organization) levels.

Reviewer: ```suggestion
By default, connected apps are configured at the master organization level, not at the business group (sub-organization) levels.
```

> If a connected app is created to act on its own behalf(client credentials), after the scopes scopes are chosen, you could choose which master ogranization/Business Groups to enable it.

Reviewer: ```suggestion
When you create a connected app to act on its own behalf using client credentials, you can choose which master organization or business groups to enable it in after you choose scopes.
```

> If a connected app is created to act on behalf of a user, what the connected app can do is based on the scopes and what organization/business groups this user has access to.

Reviewer: ```suggestion
A connected app that acts on behalf of a user receives its own access and permissions based on the scopes, organizations, and business groups to which the user has access.
```
